### DISCUSSION TITLE
MAC平台播放器讨论

### DISCUSSION THREAD

**Discussion by eko5624** [hooke007/mpv_PlayKit#96] [其它播放器]
目前看来MAC系同时支持HDR比较好和杜比视界的播放器是Infuse（可以在这里下载 https://www.torrentmac.net/）. 
当然MAC自带的Quicktime Player也支持HDR和Dolby Vision比较好，但无奈支持的格式太少，比如不支持MKV，不支持外挂字幕.

Infuse优缺点也很明显：
优点：支持HDR和Dolby Vision比较好，效果堪比Quicktime.
缺点：打开视频太慢，对外挂字幕支持不友好（有些字幕需要手动添加进去才能显示）

MPV目前在MAC平台的综合表现也很优秀。
优点：打开视频和解码速度比较快，配置自由度比较高。
缺点：硬解个别SDR片子颜色有时会过曝。HDR to SDR tone-mapping映射效果不好，不如Quicktime Player和Infuse.

难道是因为MPV的gpu-api用的是opengl，Infuse和Quicktime Player用的不是opengl?
现在苹果也建议软件开发者弃用opengl.假如某一天苹果完全放弃opengl, MPV IINA VLC这些播放器是不是在MAC平台就不能用了呢？
这样来看MPV要做的工作还有很多，比如实现Dolby Vision,优化HDR视频的色彩映射问题。

测试平台macbook pro 2017

**Comment by hooke007** [comment]
盘一下当前intel芯的遗老MAC的目前可选吧，按推荐排序
收费的 Movist Pro , infuse , Jriver Media Center , Blu-ray Player Pro

Movist 我的主观综合体验下是最好的，除了不支持dv和字幕渲染有点小问题，没啥好讲的，该有的都不错；
infuse 优势就是唯一的完全dv支持（提过它用的是metal渲染api），我不习惯它的操作，这是我的问题（
Jriver 套壳了libplacebo，（被我质疑过白嫖hassn的开发成果，之前把渲染器说成自己开发的，后来改了说明）有基础的dv支持，不过我本身不看dv无所谓这些，但是整个播放器体验实在算不上好，整个软件本身更偏向库管理/hifi？播放器功能跟附带的一样，什么外挂字幕支持啊等于没有。
Blu-ray这个看名字也知道是看原盘用的特殊播放器，但是到现在还没支持4K原盘。。。

免费的 VLC mpv（&iina）

VLC都熟，略过，它就属于那种当你切换到一个新平台，不知道选啥播放器时，找它准没啥毛病...
mpv在mac的体验我真的是一言难尽的糟糕，这里边它是唯一一个硬解码卡顿掉帧，软解烤铁板的存在。。。它自身在mac上的开发也几乎是停滞的。iina这个前端也几乎暂停了开发，虽然分支依然活跃，但是核心问题是mpv不给力（我真的不知道为什么mpv的硬解比vlc垮太多以至于不插电根本没法实际使用）

至于apple芯的，我还没换，据反馈直接用ios的那几个顶流播放器即可（

**Reply by eko5624** [reply]
我觉得MPV在Intel芯MAC上可以排第二(只看SDR片子可以排第一)。硬解码性能还是不错的。你用的哪年的macbook。掉帧的测试片子有吗，我测试试试看。

**Reply by hooke007** [reply]
几乎任何我常用的bdrip片源。其实原因想想也很简单。mbp2020（当年唯一的一款5600m）核显uhd630没有能力负载我的常用设置，用独显那就是大火炉，软解那就是intel i9 小火炉，某种意义上讲是”配置不行“。
另一个帖子提到过，这机器我甚至bootcamp切换到win10用mpv的体验比mac下用好太多......

**Reply by eko5624** [reply]
所以，mpv在mac上我只开hwdec=auto，其他滤镜和shader都不用，全靠好片源来撑～

**Reply by hooke007** [reply]
> 只开hwdec=auto

如果只这么用完全不如其它播放器了，更低功耗，更方便的界面操作，还不会卡....

**Reply by eko5624** [reply]
mpv按自己的使用习惯配置好也很方便呀，快捷键也方便。字幕支持也很好。大部分hdr的片子播放效果也能接受。硬解比mpv更低功耗，字幕支持也好的播放器还有谁？movist字幕不喜欢，iina停更，vlc臃肿，有些hdr片子的播放效果还不如mpv。

**Reply by hooke007** [reply]
你没有看懂。。。在我的机器条件下，mpv核显硬解没法用，独显硬解又热又费电。所有在windows上用mpv的优势，在我的mac下都是泡影。
